docs/levels.md

```
@@ -0,0 +1,81 @@
+`ln_levels` computes statistical levels from the time-weighted envelope:
+**L10** is the level exceeded 10 % of the time (event peaks), **L50** the median,
+**L90** the background level.
+
+```python
+from pyoctaveband import ln_levels
+
+stats = ln_levels(signal, fs, n=(10, 50, 90), weighting="A")
+print(f"LA10={stats[10]:.1f}  LA50={stats[50]:.1f}  LA90={stats[90]:.1f} dB")
+```
+
+<picture><source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/ln_levels_example_dark.png"><img src="https://raw.githubusercontent.com/jmrplens/PyOctaveBand/main/.github/images/ln_levels_example.png" width="80%"></picture>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)